tee-worker/scripts/launch.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
```

Comment:
when shall we run `launch.sh` and when `launch.py`?

Reply:
Both are fine. `launch.sh` would be more preferred for local launch & test. `launch.py` + `config.json` would have more flexibility on ports parameter inputs (`launch.sh` will just increase the port number between different worker. `config.json` can give any port number).